.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+npm-debug.log
```

Comment:
@lukevella Would you like to setup the `.editorconfig` to _your_ preferences? Instructions are here: http://editorconfig.org